tracer/src/Datadog.Trace/Ci/Processors/OriginTagTraceProcessor.cs

```diff
@@ -14,8 +14,8 @@ internal class OriginTagTraceProcessor : ITraceProcessor
     {
         private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor<OriginTagTraceProcessor>();
 
-        private readonly bool _isPartialFlushEnabled = false;
-        private readonly bool _isCiVisibilityProtocol = false;
```

Comment:
Personally, I find explicitly setting the field to the default makes it easier to read. I know Rider isn't a fan of it, but I think it reduces cognitive overhead in many cases

Reply:
Yeah for me too, honestly the whole PR is a balance of what is the most annoying to me, Rider complaining of the code style.